File: coco/src/main/java/coco/ep/m2/finalized/Add.java
package coco.ep.m2.finalized;

import coco.ep.m2.finalized.Exp;

public class Add extends Exp implements coco.ep.m2.Add<Exp> {

    protected Exp left;

    protected Exp right;

    public Add(Exp _left, Exp _right) {
        this.left = _left;
        this.right = _right;
    }

    public Exp getLeft() {
        return this.left;
    }

    public Exp getRight() {
        return this.right;
    }
}

File: coco_csharp/CoCo/EP/M2/Finalized/Add.cs
namespace EP.M2.Finalized {
    public class Add : Exp, EP.M2.Add<Exp> {
        private EP.Exp<Exp> left;
        private EP.Exp<Exp> right;

        public Add(EP.Exp<Exp> _left, EP.Exp<Exp> _right) {
            this.left = _left;
            this.right = _right;
        }

        public EP.Exp<Exp> getLeft() { return this.left; }

        public EP.Exp<Exp> getRight() { return this.right; }
    }
}